### PR TITLE
Ukrainian localization

### PR DESCRIPTION
**Pull request by iZonex**
**Added Ukrainian localization to the Mobileraker app**

**Main changes:**
- Added the `uk.yaml` file with translation keys for the Ukrainian language.
- Integrated Ukrainian localization into the app settings via `supportedLocales`.

**Details:**
- All keys were translated considering the context and specifics of the app.
- The translation was done taking into account the grammatical and stylistic features of the Ukrainian language.

### CODE REVIEW

File: assets/translations/uk.json [hunk 1]
```diff
@@ -0,0 +1,1122 @@
...
+        "body_error": "Помилка під час друку файлу: \"{}\""
+      },
+      "progress": {
+        "name": "Оновлення ходу друку - {}",
+        "desc": "Сповіщення щодо ходу друку.",
+        "title": "Хід друку {}"
+      }
+    }
+  },
+  "languages": {
```
Comment (Clon1998) [inline assets/translations/uk.json:380]: No need to add the `languages` key. The app will use the values from the `en.json` file as fallback.

**Review by Clon1998 — CHANGES_REQUESTED** [review]

File: assets/translations/uk.json (hunk 1 above)
Comment (iZonex) [reply assets/translations/uk.json:380]: Ok, I will remove :) Thank for review

**Comment by Clon1998** [comment]
Thanks for your contribution.
I will verify that everything is fine with the app later today and merge it in that case.

Just as a small heads-up, the notifications are now created and handled solely by the companion.
Therefore, it would be nice if you could also create a PR for the companion :).

[i18n](https://github.com/Clon1998/mobileraker_companion/blob/main/mobileraker/util/i18n.py) file in Companion

**Review by Clon1998 — APPROVED** [review]
LGTM

**Pull request merged**